tools/geninterop/geninterop.py

```diff
@@ -21,6 +21,11 @@
 import sysconfig
 import subprocess
 
+if sys.version_info.major > 2:
```

Comment:
We no longer support 2.x

Reply:
Travis is still using Python2 to build, though.